src/test/ui/specialization/specialization-default-methods-fail.rs

```diff
@@ -1,11 +1,11 @@
-// compile-fail
```

Comment:
Also arguably the test should be renamed now. =P (Should have mentioned earlier, sorry! ^^)